app/javascript/components/widgets/weather/weather-effect.jsx

```diff
@@ -345,64 +346,88 @@ const DrizzleLight = () => <Drizzle duration={1.8} variance={0.5} count={30} />;
+  503: [{ Type: RainIntense, props: {},idx:34}], // Very heavy
+  504: [{ Type: RainExtreme, props: {},idx:35}], // Extreme
+  511: [{ Type: RainModerate, props: {},idx:36}], // Freezing
+  520: [{ Type: RainLight, props: {},idx:37}], // Light shower
+  521: [{ Type: RainModerate, props: {},idx:38}], // Moderate shower
+  522: [{ Type: RainHeavy, props: {},idx:39}], // Heavy shower
+  531: [{ Type: RainModerate, props: {},idx:40}], // Ragged shower
+  600: [{ Type: Snow, props: { duration: 6, variance: 2, count: 20 },idx: 41} ], // Light
+  601: [{ Type: Snow, props: { duration: 4, variance: 2, count:520 },idx: 42} ], // Normal
+  602: [{ Type: Snow, props: { duration: 2, variance: 2, count: 70 },idx:43} ], // Heavy
+  611: [{ Type: Snow, props: { duration: 3, variance: 2, count: 20 },idx:44} ], // Sleet
+  612: [{ Type: Snow, props: { duration: 2, variance: 1, count: 35 },idx:45}], // Sleet
+  613: [{ Type: Snow, props: { duration: 1.5, variance: 1,count: 45 },id:46} ], // Sleet
+  615: [{ Type: Snow, props: { duration: 4, variance: 2, count: 20 },idx:47} , { Type: RainLight, props: {},idx:48}], // Light rain and snow 
+  616: [{ Type: Snow, props: { duration: 2, variance: 2, count: 30 },idx:49} , { Type: RainModerate, props: {},idx:50}], // Rain and snow
+  620: [{ Type: Snow, props: { duration: 6, variance: 1, count: 20 },idx: 51} ], // Light
+  621: [{ Type: Snow, props: { duration: 4, variance: 1, count: 20 },idx: 52} ], // Normal
+  622: [{ Type: Snow, props: { duration: 2, variance: 1, count: 20 },idx: 53} ], // Heavy
+  801: [{Type: Cloud, props:{duration: 16, variance: 1, count: 100}, idx:54}], // few cloud
+  802: [{Type: Cloud, props:{duration: 16, variance: 1, count: 4},idx:55}], // scattered cloud
+  803: [{Type: Cloud, props:{duration: 16, variance: 1, count: 6},idx:56}], // broken cloud
+  804: [{Type: Cloud, props:{duration: 16, variance: 1, count: 16},idx:57}], // overcast cloud
 }
-/* eslint-enable prettier/prettier */
 
 const WeatherEffect = ({ weather }) => {
   const {
     current: {
       weather: { id },
-    },
+    },   
   } = weather;
-
+  
+  
+  /* eslint-enable prettier/prettier */
   const Effect = weatherIdMap[id];
 
-  return Effect ? <EffectContainer>{Effect}</EffectContainer> : null;
+  return Effect ? (
+    <EffectContainer>
+      {Effect.map(({ Type, props, idx }) => {
+        if (weather.current.windSpeed <= 3.0) {
+          // eslint-disable-next-line no-param-reassign
+          props.duration += 15;
+          return <Type key={idx} {...props} />;
+        }
+        if (
+          weather.current.windSpeed >= 3.0 &&
+          weather.current.windSpeed <= 4.5
+        ) {
+          return <Type key={idx} {...props} />;
+        }
+        if (weather.current.windSpeed > 4.5) {
+          // eslint-disable-next-line no-param-reassign
+          props.duration -= 5;
+          return <Type key={idx} {...props} />;
+        }
+        return null;
```

Comment:
I think that this last conditional can be dropped because the first two conditionals make it such that this statement: `if (weather.current.windSpeed > 4.5)` will always evaluate to true

Reply:
Yeah. I could remove it.